source/helpers/error-handler/ErrorHandler.tsx

```diff
@@ -15,23 +16,34 @@ import { string } from 'prop-types';
  * @param error Error object or error message string
  * @param isFatal True if error is deemed fatal and cannot be recovered from
  */
-const boundaryErrorHandler = (error: undefined | string | Error, isFatal: boolean) => {
-  const errorMessage = typeof error === 'string' ? error : error?.message || 'unknown error';
+const boundaryErrorHandler = (
+  error: undefined | string | Error,
+  isFatal: boolean
+): void => {
+  const errorMessage =
+    typeof error === "string" ? error : error?.message || "unknown error";
 
   if (error && isFatal) {
-    console.log('boundary error', typeof error, error);
+    console.log("boundary error", typeof error, error);
 
-    if (Config.IS_STORYBOOK === 'true') {
-      Alert.alert('Something went wrong in storybook', `${errorMessage}`, [{ text: 'OK' }]);
+    if (Config.IS_STORYBOOK === "true") {
+      Alert.alert("Something went wrong in storybook", `${errorMessage}`, [
+        { text: "OK" },
+      ]);
     } else {
-      Alert.alert('Something went wrong in Mitt Helsingborg', `${errorMessage}`, [
-        {
-          text: 'Restart',
-          onPress: () => {
-            RNRestart.Restart();
+      getMonitoringService().sendError(error);
```

Comment:
Maybe we should handle non-fatal errors as well? Those are `console.error(...)` whereas fatal are uncaught exceptions.